solutions/src/main/java/ru/sberbank/school/task08/state/GameObject.java

```diff
@@ -31,8 +44,12 @@ public void setHitPoints(long hitPoints) {
 
     @Override
     public boolean equals(Object o) {
-        if (this == o) return true;
-        if (!(o instanceof GameObject)) return false;
+        if (this == o) {
+            return true;
+        }
+        if (!(o instanceof GameObject)) {
+            return false;
+        }
         GameObject that = (GameObject) o;
         return type == that.type &&
```

Comment:
&#39;&amp;&amp;&#39; should be on a new line.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1003517125>*